src/xpcc/architecture/driver/atomic/queue.hpp

```diff
@@ -67,6 +67,9 @@ namespace xpcc
 			ALWAYS_INLINE bool
 			isFull() const;
 			
+			ALWAYS_INLINE bool
+			isNotFull() const { return not isFull(); }
```

Comment:
I've noticed using `not` and other keyword versions of operators in xpcc, is this intentional? I haven't seen them used anywhere else in any C++ code, and I would think many people don't even know about these, so it might be confusing. On the other hand, I can accept that they can be more readable.